src/components/panes/QueryPane.jsx

```diff
@@ -49,22 +51,43 @@ export default class QueryPane extends PaneBase {
     }
 
     getPaneSubTitle(data) {
-        return (
-            <a key="editLink" onClick={ this.onEditClick.bind(this) }>
+        if (data.queryItem && 
+            data.queryItem.data && 
+            data.queryItem.data.organization && 
+            data.queryItem.data.organization.id == data.activeOrg) {
```

Comment:
I really love the modern JS (and TS) [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) and can't wait to leave this codebase behind. 😊

Reply:
Yeah I was thinking the same thing as I wrote this 😄